pycqed/measurement/calibration/single_qubit_gates.py

```diff
@@ -119,26 +116,35 @@ def add_amplitude_sweep_points(self, task_list=None):
                 qubit_freqs = self.sweep_points[1]['qubit_freqs'][0]
             else:
                 qubit_freqs = None
-            if qubit_freqs is not None:
-                qubits, _ = self.get_qubits(task['qb'])
+            if 'amplitude' in sweep_points[1]:
+                amplitudes = sweep_points[1]['amplitude'][0]
+            elif len(self.sweep_points) >= 2 and \
+                    'amplitude' in self.sweep_points[1]:
+                amplitudes = self.sweep_points[1]['amplitude'][0]
```

Comment:
My favourite comment to make: do we want this here and in lines 120, 122, or should we use the get methods? :p

Reply:
I can make the change if we agree.

Reply:
I now access it via __getitem__, which is the cleanest & shortest solution IMO.

Reply:
👍